CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.20)
+project(catalyst)
+
+set(LOGO [=[
+░█▀▀░█▀█░▀█▀░█▀█░█░░░█░█░█▀▀░▀█▀
+░█░░░█▀█░░█░░█▀█░█░░░░█░░▀▀█░░█░
+░▀▀▀░▀░▀░░▀░░▀░▀░▀▀▀░░▀░░▀▀▀░░▀░
```

Comment:
😃

Reply:
Very important it gets the `toilet -f pagga "catalyst"` treatment